src/P/EitherT.hs

```diff
@@ -1,5 +1,12 @@
 module P.EitherT (
-    firstEitherT
+    EitherT(..)
+  , eitherT
+  , bimapEitherT
+  , left
+  , hoistEither
+  , mapEitherT
```

Comment:
@charleso I'm not opposed, but i think i dont want to export the entire `EitherT` module, the functions listed are the ones I am certain are useful and not redundant (for example `right` is redundant), and unless i am going to switch to `hiding` the functions we dont want (which is less than the number of functions we do want) theres no difference.

So assuming we want to switch to a hiding list:

I would have a question about which functions we want to hide if any.  

In particular i want to ask if we want to hide [`bracketEitherT`] (https://hackage.haskell.org/package/either-4.4.1/docs/Control-Monad-Trans-Either.html#v:bracketEitherT) cos it looks pretty useless to me.

/cc @markhibberd 

-------

Also note that the advantage of listing the functions we do want is that if someone comes along and adds cruft to `either` we dont want, it wont get implicitly added to `p` when we bump versions.  Seeing as we import P into all of our projects, I personally dont mind listing the exports rather than the hides even iof they are longer since we only have to have that long list in one place